web/viewer.js

```diff
@@ -996,7 +980,7 @@ var PDFViewerApplication = {
     if (this.pdfViewer.currentScale === UNKNOWN_SCALE) {
```

Comment:
I still think you need this change though, but it seems to have been lost in the latest patch :-)